feat(service-registry rule): add base and enable command (#1418)
redhat-developer/app-services-cli: docs/commands/rhoas_service-registry_rule.md

## rhoas service-registry rule

> Manage rules in a Service Registry instance

Reviewer: ```suggestion
Manage artifact rules in a Service Registry instance
```

### Synopsis

> Manage validity and compatibility rules for a Service Registry instance and artifacts.

Reviewer: ```suggestion
Set of commands for managing validity and compatibility rules for an entire Service Registry instance or individual for each artifact.
```

> Service Registry rules govern how registry content can evolve over time.

Reviewer: Left comments in wrong file. Sorry for that

Rules can be configured as global rules for entire Service Registry instances or specific artifacts.

### Examples

```

> ## Enable global compatibility rule for a specific Service Registry instance

Reviewer: I think that "specific registry instance" sounds confusing. I would use "globally for artifacts" as more precise form
```suggestion
## Enable global compatibility rule for a specific Service Registry instance
```

Author: The service registry docs uses the term `global rules`. Does "Enable global compatibility rules for artifacts of current service registry instance" sounds good?

Reviewer: yes. global rules sound good
